e3-redesign/src/donate.html

```diff
@@ -25,39 +25,37 @@ <h2 class="content__title content__title--form">Contact Information</h2>
                 </div>
                 <div class="form__field form__contact__fname">
                     <label class="form__label" for="first-name">First Name </label>
-                    <input type="text" class="text" name="firstName" id="firstName" placeholder="Salvador" />
+                    <input type="text" class="text" name="first-name" id="first-name" placeholder="Salvador" />
                 </div>
                 <div class="form__field form__contact__mname">
-                    <label class="form__label" for="middle-name">Middle Name <span class="form--optional">(optional)</span></label>
-                    <input type="text" class="text" name="middleName" id="middleName" placeholder="Helicopter" />
+                    <label class="form__label" for="middle-name">Middle Name <span class="form__optional">(optional)</span></label>
+                    <input type="text" class="text" name="middle-man" id="middle-name" placeholder="Helicopter" />
                 </div>
                 <div class="form__field form__contact__lname">
                     <label class="form__label" for="last-name">Last Name</label>
-                    <input type="text" class="text" name="lastName" id="lastName" placeholder="Dali" />
+                    <input type="text" class="text" name="last-name" id="last-name" placeholder="Dali" />
                 </div>
                 <div class="form__field form__contact__address">
                     <label class="form__label" for="address">Address</label>
                     <input type="text" name="address" class="address" id="address" cols="32" rows="3" placeholder="800 Key Hwy"></input>
```

Comment:
Here, you've muddied up some of the attributes for [`input`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) and [`textarea`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea).
This would be valid for a text `input`:
```suggestion
                     <input type="text" name="address" class="address" id="address" placeholder="800 Key Hwy">
```
and this would be valid for `textarea`:
```suggestion
                     <textarea name="address" class="address" id="address" cols="32" rows="3" placeholder="800 Key Hwy"></textarea>
```